cocos2d/core/event-manager/CCTouch.js

```diff
@@ -35,6 +35,7 @@
  */
 cc.Touch = function (x, y, id) {
     this._lastModified = 0;
+    this.windowId = undefined;
```

Comment:
Is it possible to make this variable private?

Reply:
Why need to add private tag in js code?

Reply:
> Just like _lastModified, it is private and obtains the value through a function. We agree that variables starting with an underscore are private

ok, I have modified the name to _currentTouchId

Reply:
If it is private, should the value be obtained through member methods?

Reply:
> If it is private, should the value be obtained through member methods?

fixed.